src/libraries/Microsoft.Extensions.Hosting/src/Host.cs

```diff
@@ -31,7 +28,6 @@ public static class Host
         ///   </list>
         /// </remarks>
         /// <returns>The initialized <see cref="IHostBuilder"/>.</returns>
-        [RequiresDynamicCode(RequiresDynamicCodeMessage)]
```

Comment:
Nice. Is hosting the only libraries that used DI and bubbled up `RequiresDynamicCode`?

Reply:
No - Logging did as well.